RestApi/Sepes.RestApi/Services/SepesDb.cs

```diff
@@ -106,7 +106,7 @@ public async Task<int> createStudy(string studyName, int[] userIds, int[] datase
                 command.Parameters.AddWithValue("@podName", name);
                 command.Parameters.AddWithValue("@studyID", studyId);
                 var id = Convert.ToUInt16(await command.ExecuteScalarAsync());
-                return new Pod(id, name, studyId);
+                return new Pod(id, name, studyId, false, null, null, null, null, null);
```

Comment:
Do not like the use off null here. Null have a tendency blow up code that do not handle the null case explicitly.